pallets/runtime/src/multisig.rs

```diff
@@ -530,6 +532,10 @@ impl<T: Trait> Module<T> {
         }
         <MultiSigSignsRequired<T>>::insert(&account_id, &sigs_required);
         <MultiSigCreator<T>>::insert(&account_id, &signer_did);
+        <identity::KeyToIdentityIds>::insert(
+            AccountKey::try_from(account_id.encode())?,
+            LinkedKeyInfo::Unique(signer_did),
```

Comment:
minor, but should the `signer_did` variable be called `sender_did` instead (possible confusion with the `signers` variable).